### DISCUSSION TITLE
QUDI recommended OS

### DISCUSSION THREAD

**Discussion by AItah** [Ulm-IQO/qudi-core#76] [Q&A] [answered]
Hi all,

Is there a recommended OS (Windows/Ubuntu/ ...) which is recommended for QUDI?

BR,
Amir

**Comment by Geegee** [comment, marked as answer]
It should both work with Linux and Windows but here in Ulm at the Institute for Quantum Optics
we mainly use Windows so it is probably better maintained and supported.

-- 
 Sent with Tutanota; enjoy secure & ad-free emails: 
 https://tutanota.com


Oct 29, 2023, 08:14 by ***@***.***:

**Reply by Neverhorst** [reply]
Some even used qudi on macOS but Linux/Unix and Mac users never really gave feedback upstream on the matter (i.e. if all dependencies work fine, what did they do to make it work etc.). So if you want to use qudi in a non-Windows environment, we would highly appreciate your contribution to improve support for these platforms.
